client/utils/rest.go

```diff
@@ -37,9 +44,21 @@ func HasDryRunArg(r *http.Request) bool { return urlQueryHasArg(r.URL, queryArgD
 // HasGenerateOnlyArg returns whether a URL's query "generate-only" parameter is set to "true".
 func HasGenerateOnlyArg(r *http.Request) bool { return urlQueryHasArg(r.URL, queryArgGenerateOnly) }
 
-// ParseFloat64OrReturnBadRequest converts s to a float64 value. It returns a default
-// value if the string is empty. Write
-func ParseFloat64OrReturnBadRequest(w http.ResponseWriter, s string, defaultIfEmpty float64) (n float64, ok bool) {
+// ParseInt64OrReturnBadRequest converts s to a float64 value.
```

Comment:
`float64` -> `int64`

Reply:
That's really a `float64`. It's being used to parse HTTP requests gas adjustment fields, which come as strings.

Reply:
But the function returns an `int64`

Reply:
Oh right yes, please ignore my comment then.